.travis.yml

```diff
@@ -29,4 +34,40 @@ env:
     - TEST_DIR=core
     - TEST_DIR=util
 
+after_success: |
+  pwd &&
+  wget https://github.com/percytheprefect/kcov/archive/master.tar.gz &&
+  tar xzf master.tar.gz &&
+  cd kcov-master &&
+  mkdir build &&
+  cd build &&
+  cmake .. &&
+  make &&
+  sudo make install &&
+  cd ../.. &&
+  rm -rf kcov-master &&
+  cargo test --no-run &&
+  cd ../
+  pwd &&
+  kcov --exclude-pattern=/.cargo,/usr/lib --verify target/cov target/debug/grin_core* && 
```

Comment:
@percytheprefect thanks for adding the other directories.

Are you sure these exist?  I have been trying to follow [the tutorial you linked](http://sunjay.ca/2016/07/25/rust-code-coverage) locally: after running `cargo test --no-run`, my `target/debug` dir looks like this:
![](https://puu.sh/yODGQ/b542e33baf.png)

I think this is why none of our builds are uploading coverage. When we run
`kcov --exclude-pattern=/.cargo,/usr/lib --verify target/cov target/debug/grin-chain*`, there is no `grin-chain` executable.  Travis should behave the same.  I also tried a [few times](https://github.com/sdtsui/grin/pulls) with different configurations on my fork.